instrumented/integration/src/androidTest/kotlin/com/datadog/android/sdk/integration/rum/ActivityTrackingTest.kt

```diff
@@ -11,8 +11,10 @@ import androidx.test.platform.app.InstrumentationRegistry
 import com.datadog.android.sdk.rules.RumMockServerActivityTestRule
 
 internal abstract class ActivityTrackingTest :
-    RumTest<ActivityTrackingPlaygroundActivity,
-        RumMockServerActivityTestRule<ActivityTrackingPlaygroundActivity>>() {
+    RumTest<
```

Comment:
very strange formatting applied by `ktlint`

Reply:
Yeah essentially class with generics, when spread multiline, want to be as readable as arrays `[]` or blocks `{}`.